index.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const puppeteer = require('puppeteer');
+const express = require('express');
+
+const main = async (answers) => {
+    try {
+        const port = 5001;
+        const destPath = path.join(process.cwd(), answers.destinationPath);
+        const fullResumePath = `http://localhost:${port}/${answers.sourcePath}`;
+        let destFilename = answers.fileName;
+        const app = express();
+        app.use(express.static(path.join(process.cwd())));
+        const server = app.listen(port, async () => {
+            const browser = await puppeteer.launch();
+            const page = await browser.newPage();
+            if (!destFilename.endsWith('.pdf')) {
+                destFilename += '.pdf';
+            }
+            await page.goto(fullResumePath, { waitUntil: 'networkidle0' });
+            await page.pdf({

```

Comment:
Would there be a way to configure custom arguments for this in the .html-to-pdf.json file? My particular resume uses very tiny margins in order to print correctly. I would also assume that other people would have different needs as well (different paper sizes, landscape, etc)

Reply:
Added prompts and logic to handle:
* Margins
* Landscape vs Portrait
* Paper format (Letter, A4, etc)
* Print background graphics